modules/RTC/ScreenObtainer.js

```diff
@@ -5,7 +5,10 @@ import browser from '../browser';
 
 const logger = require('jitsi-meet-logger').getLogger(__filename);
 
-let gumFunction = null;
+/**
+ * The default frame rate for Screen Sharing.
+ */
+const SS_DEFAULT_FRAME_RATE = 5;
```

Comment:
The import [here](https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/xmpp/JingleSessionPC.js#L12) needs to be re-adjusted because of the above change.

Reply:
👍